src/style/index.css

```diff
@@ -1,5 +1,20 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600;700&family=Roboto+Slab:wght@200;300;700&display=swap');
+
 body {
+  width: 100%;
   margin: 0;
   padding: 0;
-  font-family: sans-serif;
+  font-family: Montserrat, sans-serif;
+  background-color: lightgray;
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+}
```

Comment:
- [ ] <b>REQUIRED:</b> There are still few things we need to fix to make our app look more like the provided design. Kindly review the following issues and make sure to fix them.

| Original      | Yours |
  | ----------- | ----------- |
| ![80047697-0c3f6280-8506-11ea-9416-a4f1800e2329](https://user-images.githubusercontent.com/45217364/130871998-88863e2f-c823-4b7b-a5c4-20162973ff21.png) | ![scrnli_3_6_2022_1-29-23 AM](https://user-images.githubusercontent.com/45217364/156903116-c39e3349-56c0-4bb0-b2ac-f69b787c8c53.png) |

  - The app is not fullscreen, it is more like a card in the center of the screen.
  - As the project requires, make sure to follow vertical and horizontal alignment just like there are in mockup design.
  - And also, try to make sure to use the same exact colors as the mockup design.